draft-ietf-tls-tls13.md

```diff
@@ -3752,9 +3752,10 @@ New Alert values are assigned by IANA as described in {{iana-considerations}}.
 
 #  Cryptographic Computations
 
-In order to begin connection protection, the TLS Record Protocol
-requires specification of a suite of algorithms, a master secret, and
-the client and server random values.
+In order to authenticate and to start encrypting the exchange, the TLS Record
+Protocol needs to derive a suite of secrets via the key schedule. To perform
+these operations, a peer makes use of observed handshake messages, an optional
+pre-shared secret and an optional output of the (EC)DHE key exchange.
```

Comment:
I would prefer "at least one of a pre-shared secret and the output of an (EC)DHE key exchange", since they are only optional if the other one is present.